DotaStats/PlayersModule/PlayersModule.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+class PlayersModule {
+    private let view: PlayersModuleInput
+    private let presenter: PlayersModuleOutput
+    private let networkService: NetworkSeviceProtocol
+    
+    init() {
+        networkService = NetworkService()
+        presenter = PlayersModulePresenter(networkService: networkService)
```

Comment:
Лучше сделать приблизительно так
```suggestion
    init(networkService: NetworkServiceProtocol) {
        presenter = PlayersModulePresenter(networkService: networkService)
```